style.css

```diff
@@ -3,571 +3,1197 @@ html body {
+    border-radius: 8px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+  }
+
+  .bigTextBox {
+    height: 170px;
+    width: 327px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 8px;
+    resize: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+  }
+
+  #submit-btn {
+    background: none;
+    border: none;
+    padding: 0;
+    font: inherit;
+    cursor: pointer;
+    outline: inherit;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    text-decoration: none;
+  }
+
+  .submitbtn-area {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 12px 16px 12px 16px;
+    padding-top: 20px;
+    width: 140px;
+    height: 30px;
+    background: #fff;
+    border-radius: 8px;
+    margin: 24px;
+    margin-right: 100px;
+  }
+
+  .TextBox-personal-style {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 20px;
+    color: #172b4d;
+  }
+
+  .TextBox-info-style {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+  }
+
+  .Shape-line {
+    margin-left: 120px;
+    margin-bottom: 8px;
+    height: 5px;
+    width: 134px;
+    left: 120px;
+    top: 21px;
+    border-radius: 10px;
+  }
+
+  input {
+    padding: 10px;
+    padding-right: 0;
+  }
+
+  input:focus {
+    outline: none !important;
+    border-color: #6070ff;
+    box-shadow: 0 0 10px #0d99ff;
+  }
+
+  textarea {
+    padding: 10px;
+    padding-right: 0;
+    flex-basis: 100%;
+  }
+
+  textarea:focus {
+    outline: none !important;
+    border-color: #6070ff;
+    box-shadow: 0 0 10px #0d99ff;
+  }
+
+  .resumebtn-area:hover {
+    outline: none !important;
+    border-color: #6070ff;
+    box-shadow: 0 0 4px #0d99ff;
+  }
+
+  .submitbtn-area:hover {
+    outline: none !important;
+    border-color: #6070ff;
+    box-shadow: 0 0 4px #fff;
+  }
+
+  .btn-area:hover {
+    outline: none !important;
+    border-color: #6070ff;
+    box-shadow: 0 0 4px #0d99ff;
+  }
+
+  #submit-btn:active {
+    color: #fff;
+  }
+
+  .project-btn:active {
+    color: #fff;
+  }
+
+  #resume-btn:active {
+    color: #fff;
+  }
+
+  .styleLine {
+    margin-left: 200px;
+  }
+}
+
+/* *************************************************************** */
+
+/* *************************************************************** */
+
+/* ************************************************************** */
+
+@media only screen and (min-width: 765px) {
+  .mobile-version {
+    display: none;
+  }
```

Comment:
- [ ] For the desktop version, please ensure that the `min-width` is between 768px or wider 👍